src/main/resources/openapi.json

```diff
@@ -1066,30 +1066,41 @@
     },
     "components": {
         "schemas": {
+            "ConsumerRecord": {
+                "example": {
+                    "key": "key1",
+                    "offset": 2,
+                    "partition": 0,
+                    "topic": "topic",
+                    "value": "value1"
+                },
+                "properties": {
+                    "key": {
+                        "type": "string"
+                    },
+                    "offset": {
+                        "format": "int32",
```

Comment:
can you check that format is int64 for other definitions having offset please? In that case we should have int64 here as well.